apps/admin/backend/src/tabulation/utils.ts

```diff
@@ -30,3 +30,10 @@ export function replacePartyIdFilter(
     batchIds: filter.batchIds,
   };
 }
+
+/**
+ * Tests whether CVR votes are empty.
+ */
+export function isBlankVotes(votes: Tabulation.Votes): boolean {
```

Comment:
opt: maybe something along the lines of `isBlankBallotCard` or `isBlankCard` instead?

Reply:
that sounds better, i sincerely appreciate the naming suggestions 🤖